inc/Addon/FacebookTracking/Subscriber.php

```diff
@@ -11,7 +12,7 @@
  * @since  3.2
  * @author Grégory Viguier
```

Comment:
Remove all `@author` instances in this class.

Reply:
https://github.com/wp-media/wp-rocket/pull/2616/commits/dd041589183f9e6ea56fdbce81270869d5f90c2e